### PR TITLE
added chinese version of README

### PR DESCRIPTION
**Pull request by junqinghua**
- [x] Because this PR includes a **new feature**, the change was previously discussed on an Issue or with someone from the team.

---

### CODE REVIEW

File: ChineseVersionDocs/README.cn.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+![](https://user-images.githubusercontent.com/176499/96893278-ebc67580-1460-11eb-9530-d5df3a3d65d0.png) [![NPM Package](https://img.shields.io/npm/v/hardhat.svg?style=flat-square)](https://www.npmjs.org/package/hardhat) [![GitPOAP Badge](https://public-api.gitpoap.io/v1/repo/NomicFoundation/hardhat/badge)](https://www.gitpoap.io/gh/NomicFoundation/hardhat)
```
Comment (fvictorio) [inline ChineseVersionDocs/README.cn.md:1]: I think we should use a slightly different directory structure. Something like `/translations/chinese/hardhat-core/README.md` and `/translations/chinese/docs/README.md` maybe?

File: docs/README.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # Hardhat documentation website
 
+[中文翻译文档](../ChineseVersionDocs/README_doc.cn.md)
```
Comment (fvictorio) [inline docs/README.md:3]:
```suggestion
Translations of this document:
- [中文翻译文档](../ChineseVersionDocs/README_doc.cn.md)
```

File: packages/hardhat-core/README.md [hunk 3]
```diff
@@ -8,6 +8,8 @@ Built by the [Nomic Foundation](https://nomic.foundation/) for the Ethereum comm
 
 Join our [Hardhat Support Discord server](https://hardhat.org/discord) to stay up to date on new releases, plugins and tutorials.
 
+[中文翻译文档](./ChineseVersionDocs/README.cn.md)
```
Comment (fvictorio) [inline packages/hardhat-core/README.md:11]:
```suggestion
Translations of this document:
- [中文翻译文档](./ChineseVersionDocs/README.cn.md)
```

**Review by fvictorio — COMMENTED** [review]
Thanks a lot for sending this @junqinghua! I left some comments.

File: ChineseVersionDocs/README.cn.md (hunk 1 above)
Comment (junqinghua) [reply ChineseVersionDocs/README.cn.md:1]: That will be a much better directory structure. I will work on it soon.

**Comment by fvictorio** [comment]
Hey @junqinghua, are you still interested in working on this?

**Comment by schaable** [comment]
Closing due to lack of activity. @junqinghua, feel free to send a new PR if you want to add it in the future.

**Pull request closed without merging**